config.json

```diff
@@ -411,7 +411,7 @@
            "CmakeBuildType": "default",
            "HostOs": "default",
            "ProcessorCount": "default",
-           "AdditionalArgs": "default",
```

Comment:
Have you confirmed that build.{sh|cmd} are passing the flag correctly all the way to the native component build?